bftengine/src/bftengine/TimeServiceResPageClient.cpp

```diff
@@ -0,0 +1,40 @@
+// Concord
+//
+// Copyright (c) 2018-2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").  You may not use this product except in
+// compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license terms. Your use of
+// these subcomponents is subject to the terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+
+#include "TimeServiceResPageClient.hpp"
+
+namespace {
+constexpr auto RESERVED_PAGE_ID = uint32_t{0};
```

Comment:
Funny, you could define it just as constexpr uint32_t RESERVED_PAGE_ID = 0;

Reply:
@yuliasherman, but of course!
I kinda like the AAA (almost always auto) approach introduced by Herb Sutter:
https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/ :)